src/app/loadout-builder/filter/FilterBuilds.tsx

```diff
@@ -37,6 +42,85 @@ export default function FilterBuilds({
   const workingStatRanges =
     statRanges || _.mapValues(statHashes, () => ({ min: 0, max: 10, ignored: false }));
 
+  const legendaryShardDef = defs?.InventoryItem.get(UpgradeMaterialHashes.legendaryShard);
+  const enhancementPrismDef = defs?.InventoryItem.get(UpgradeMaterialHashes.enhancementPrism);
+  const ascendantShardDef = defs?.InventoryItem.get(UpgradeMaterialHashes.ascendantShard);
+  const legendaryShardIcon = legendaryShardDef?.displayProperties.icon;
+  const enhancementPrismIcon = enhancementPrismDef?.displayProperties.icon;
+  const ascendantShardIcon = ascendantShardDef?.displayProperties.icon;
+  const ascendantShardName = ascendantShardDef?.displayProperties.name || 'Ascendant Shard';
```

Comment:
Just using an english string as a fallback for these names but I never actually expect it to get used.